modules/neovim/mappings/options.nix

```diff
@@ -31,59 +31,72 @@
     options =
       mapConfigOptions
       // {
+        key = mkOption {
+          type = str;
+          description = ''
+            Key that triggers this keybind.
+          '';
+        };
         mode = mkOption {
           type = either str (listOf str);
           description = ''
             The short-name of the mode to set the keymapping for. Passing an empty string is the equivalent of `:map`.
 
             See `:help map-modes` for a list of modes.
           '';
+          example = ''"nvc" for normal, visual and command mode'';
```

Comment:
nvc isn't a real short name from looking at the docs?

Reply:
oof, I thought nvim supports this, turns out no